lambdas/authorizer/package.json

```diff
@@ -3,18 +3,28 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "build": "tsc --outDir ./.build src/index.ts",
+    "build": "esbuild src/index.ts --bundle --outdir=.build --format=cjs --platform=node",
     "test:unit": "jest",
     "typecheck": "tsc --listFiles --noEmit"
   },
   "devDependencies": {
     "@swc/core": "^1.9.2",
     "@swc/jest": "^0.2.37",
     "@tsconfig/node20": "^20.1.4",
+    "@types/aws-lambda": "^8.10.145",
     "@types/jest": "^29.5.14",
+    "@types/jsonwebtoken": "^9.0.7",
     "aws-lambda": "^1.0.7",
     "jest": "^29.7.0",
     "jest-mock-extended": "^3.0.7",
     "typescript": "^5.5.4"
+  },
+  "dependencies": {
+    "@aws-sdk/client-cognito-identity-provider": "^3.693.0",
+    "esbuild": "^0.24.0",
```

Comment:
small thing - esbuild should be a dev dependency

Reply:
updated